### PR TITLE
Add website redesign

### PR DESCRIPTION
**Pull request by mmejia02**
# Changes

Added Website Re-design files

-
-
-

/kind <kind>

Fixes #

**Release Note**

```release-note

```

**Docs**

```docs

```

### CODE REVIEW

**Comment by Leo6Leo** [comment]
Heyy @mmejia02 ! You may need to complete the EasyCLA to proceed with merging the PR! 

Hope the following instruction helps! Lmk if you have any issues with accessing it.
https://knative.dev/blog/articles/getting-started-blog-p2/#cla

**Comment by Cali0707** [comment]
/cc @knative/steering-committee 

This is the first set of proposed website changes which we hope are able to be implemented in the short term, before we try to do a more comprehensive set of design changes. Would you be able to:
1. Review the design changes in this PR and let us know if you are happy with them
2. Comment on this process - we weren't sure what the best way to present designs to the SC would be, so we would love feedback on if this works for you and how we could make it better.

Thanks!

File: designs/Website_V1.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# List of first re-design fixes:
```
Comment (aliok) [inline designs/Website_V1.md:1]: Can we have a link to the png images in this file? Otherwise the images are just staying orphan.

**Comment by aliok** [comment]
> Comment on this process - we weren't sure what the best way to present designs to the SC would be, so we would love feedback on if this works for you and how we could make it better.

I think overall, the process is good. Suggesting changes with reasons is a good approach. I also like that UX WG is creating PRs instead of Google Docs (good to have commits by members).
I wish GitHub was allowing comments for an image region, but if people want to comment about the images, they can download the image, manually annotate the region and upload it again :)

If we're going to have more design changes (or, structure changes) in the website, it might be useful to document the process for change requests (I wouldn't do it now though).

> Review the design changes in this PR and let us know if you are happy with them

I personally am happy with the proposed changes. Let see what folks from @knative/steering-committee will say.
Image for the new design looks absolutely much much better, thanks!


Also, what's the way forward after, say, this PR is merged?

**Comment by Cali0707** [comment]
> Also, what's the way forward after, say, this PR is merged?

@aliok the plan will be for UX WG to split this up into issues which we will open on the docs repository. Ideally, when people open PRs to resolve those issues, we can cc one of the designers for a review, to ensure that the change looks correct

**Comment by nainaz** [comment]
+1 for changes

**Pull request merged**